Steps/ProfilePageStepDefinitions.cs

```diff
@@ -0,0 +1,53 @@
+using OpenQA.Selenium;
+using QA_Mars_OnboardingTaskSpecflow.Pages;
+using TechTalk.SpecFlow;
+using Xunit;
+
+namespace QA_Mars_OnboardingTaskSpecflow.Steps
+{
+    [Binding]
+    public class ProfilePageStepDefinitions
+    {
+        private readonly IWebDriver driver;
+        private readonly ProfilePage profilePage;
+
+        public ProfilePageStepDefinitions(IWebDriver driver)
+        {
+            this.driver = driver;
+            profilePage = new ProfilePage(driver);
+        }
+
```

Comment:
you also need to think about state State

example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always
you can assume that test execution order is not always guaranteed
imagine if you run the tests  in parallel mode the all tests will run at the same time
so you need to also make sure you do not wipe out other tests data
hence why when you remove data the best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2

So to sum up you should delete all data if there is any before the start of any test and then after each test you should delete the data that a particular test is adding!
This is ensure that upon staring the test pack you have reset state and upon finishing each test deletes their data that they added!

Reply:
Thank you for your feedback! This has been fixed.

Reply:
can you explain how did you implement this please

Reply:
Hi Aman,
Thank you for reviewing the task. According to your this review feedback, I incorporated and automated another test scenario to identify existing/duplicate records and deleted records via a separate scenario. If I misunderstood anything, please let me know. I highly appreciate your support. Thank you! ☺️